tests/conftest.py

```diff
@@ -227,6 +231,87 @@ def assessment_object(group_object, page_object, campaign_object):
     )
 
 
+@pytest.fixture
+def multiple_campaign_object(campaign_object):
+    """Return list of campaign objects."""
+    campaigns = list()
+
+    for x in range(1, 8):
+        campaigns.append(
+            Campaign(
+                name=f"RVXXX1-C{x}",
+                launch_date=f"01/0{x}/2025 13:00",
+                completed_date=f"01/0{x}/2025 14:00",
+                url=f"http://bad.domain/camp{x}",
+                group_name="RVXXX1-G1",
+                template=template_object,
+                smtp=smtp_object,
+            )
+        )
+
+    # Make a campaign from a different assessment.
+    campaigns[6].name = "RVXXX2-C7"
+
+    return campaigns
+
+
+@pytest.fixture
+def multiple_gophish_group_object():
+    """Return list of GoPhish group objects."""
+    groups = list()
+
+    for x in range(1, 3):
+        groups.append(
+            GoPhish_Group(
+                group_id={x},
+                name=f"RVXXX1-G{x}",
+                targets=[
+                    GoPhish_User(
+                        first_name="Jane",
+                        last_name="Smith",
+                        email=f"jane.smith{x}@domain.tld",
+                        position="IT",
+                    ),
+                    GoPhish_User(
+                        first_name="John",
+                        last_name="Doe",
+                        email=f"john.doe{x}@domain.tld",
+                        position="HR",
+                    ),
+                ],
+            )
+        )
+
+    return groups
+
+
+@pytest.fixture
+def database_user_json():
+    """Return a User JSON with 4 emails matching the database."""
+    return [
+        {
+            # jane.smith1@domain.tld
+            "id": "8d2b7505ad58056cf54b94ad17e6755934738e78125998bd0e629306443ce10e",
```

Comment:
Rather than putting in each hash here, it's a bit cleaner to show where it came from.  I'd change it to something like this:
```
users = list()
for (email, position) in [ 
    ("jane.smith1@domain.tld", "IT"), 
    ("john.doe1@domain.tld", "HR"), 
    ("jane.smith2@domain.tld", "IT"), 
    ("john.doe2@domain.tld", "HR") 
    ]: 
    users.append( 
        { 
        "id": hashlib.sha256(email.encode("utf-8")).hexdigest(), 
        "customer_defined_labels": {"RVXXX1": [position]},
        }
    )

return users
```

Reply:
Changed to this expanded version in commit 0605cb6.

Reply:
Now that you are replacing `users` with `targets` throughout this code, you should also rename the `users` variable in `email_target_json` to `targets`.

Reply:
I was on the fence about making this change but held off as it matched `export_targets` in `gophish-export`. In this c4f7936 I changed both to reference `targets` as opposed to `users` as it makes things more uniform. 

In `export_targets` that required the following changes:
`targets` --> `raw_targets`
`target` --> `target`
`users` --> `targets`
`user` --> `target`

Reply:
Yes, I think this reads much better now. 👍